rapids-cmake/cpm/versions.json

```
@@ -11,17 +11,10 @@
       "git_tag": "097aa718f25d44315cadb80b407144ad455ee4f9"
     },
     "CCCL": {
-      "version": "2.7.0",
+      "version": "2.8.2",
       "git_shallow": false,
       "git_url": "https://github.com/NVIDIA/cccl.git",
-      "git_tag": "v${version}",
-      "patches": [
-        {
-          "file": "cccl/backport-suppress-execution-checks.patch",
-          "issue": "backport suppression for execution types for vocabulary types https://github.com/NVIDIA/cccl/pull/3578",
-          "fixed_in": "3.0.0"
-        }
-      ]
+      "git_tag": "v2.8.2-rc0"
```

2.8.2 is now released.
```suggestion
      "git_tag": "v${version}"
```